samples/automl/automlNaturalLanguageDataset.js

```diff
@@ -0,0 +1,399 @@
+/**
+ * Copyright 2018, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This application demonstrates how to perform basic operations on dataset
+ * with the Google AutoML Natural Language API.
+ *
+ * For more information, see the documentation at
+ * https://cloud.google.com/natural-language/automl/docs/
+ */
+
+`use strict`;
+
+/* eslint node/no-extraneous-require: off */
+
+function createDataset(projectId, computeRegion, datasetName, multilabel) {
+  // [START automl_natural_language_createDataset]
+  const automl = require(`@google-cloud/automl`);
+
+  const client = new automl.v1beta1.AutoMlClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = `The GCLOUD_PROJECT string, e.g. "my-gcloud-project"`;
+  // const computeRegion = `region-name, e.g. "us-central1"`;
+  // const datasetName = `name of the dataset to create, e.g. “myDataset”`;
+  // const modelName = `Name of the model, e.g. "myModel"`;
+
+  // A resource that represents Google Cloud Platform location.
+  const projectLocation = client.locationPath(projectId, computeRegion);
+
+  // Classification type is assigned based on multilabel value.
+  var classificationType = `MULTICLASS`;
+  if (multilabel) {
+    classificationType = `MULTILABEL`;
+  }
+
+  // Specify the text classification type for the dataset.
+  const datasetMetadata = {
+    classificationType: classificationType,
+  };
+
+  // Set dataset name and metadata.
+  const myDataset = {
+    displayName: datasetName,
+    textClassificationDatasetMetadata: datasetMetadata,
+  };
+
+  // Create a dataset with the dataset metadata in the region.
+  client
+    .createDataset({parent: projectLocation, dataset: myDataset})
+    .then(responses => {
+      const dataset = responses[0];
+
+      // Display the dataset information.
+      console.log(`Dataset name: ${dataset.name}`);

```

Comment:
Can we put these print statements in a helper function common to **this file**? e.g:
```javascript
const _printDataset = (dataset) => {
  ...
}
```